chapters/en/Unit 3 - Vision Transformers/Dinat Transformer.mdx

```diff
@@ -0,0 +1,66 @@
+# Dilated Neighborhood Attention Transformer (DINAT)
+
+![DINAT Architecture Diagram](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/dinat_images/dina_comparison.png)
+## Overview of Architecture
+
+The Dilated Neighborhood Attention Transformer (DiNAT) is a vision transformer that builds upon the Neighborhood Attention (NA) mechanism. DiNA extends NA's local attention by introducing a sparse global attention that captures more extensive global context without additional computational overhead. This combination powers DiNAT to maintain locality, preserve translational equivariance, and exponentially expand the receptive field. DiNAT's architecture gradually changes dilation through the model, optimizing the extension of receptive fields for effective feature learning. With these enhancements, DiNAT can outperform strong baseline architectures such as ConvNeXt.
+
+## Core of DiNAT: Neighborhood Attention
+
+DINAT is based on the Neighborhood Attention (NA) architecture which is an attention mechanism specifically designed for computer vision tasks, aiming to efficiently capture relationships between pixels in an image. In a simple analogy, imagine you have an image, and each pixel in that image needs to understand and focus on its nearby pixels to make sense of the entire picture. Let's examine the key features of NA:
+
+* **Local Relationships**: NA captures local relationships, allowing each pixel to consider information from its immediate surroundings. This is similar to how we might understand a scene by looking at the objects closest to us first before considering the entire view.
+
+* **Efficiency**: NA is designed to be efficient in processing images. It's like giving each pixel a small window or scope to look around, rather than having it examine the entire picture. This helps in reducing computational complexity and memory usage, making it more feasible for large-scale image processing tasks.
```

Comment:
```suggestion
* **Efficiency**: NA is designed to be efficient in processing images. It's like giving each pixel a small window or scope to look around, rather than having it examine the entire image. This helps in reducing computational complexity and memory usage, making it more feasible for large-scale image processing tasks.
```